manifests/config.pp

```diff
@@ -1,16 +1,31 @@
-class solr5::config inherits solr5 {
+class solr5::config (
+  $solr_data_dir,
+  $package_target_dir,
+  $solr_archive_file_name,
+  $init_config,
+  $solr_user,
+  $solr_port
+)
```

Comment:
Not consistent to other class definitions in the PR. One has the curly brace on the same line, the other has a space in between. Please pick one style